harvardsettings_pi.php

```diff
@@ -369,9 +375,10 @@ public function get_pds_emails($url) {
             else {
                 $name = $person->names[0]->firstName . " " . $person->names[0]->lastName;
             }
+            $email = $person->loginName ? $person->loginName : $person->emails[0]->email;
```

Comment:
If a person has multiple email addresses, is the first email always the "official" email address? My assumption is yes, but I'm not sure if that's really true. I believe there's an `officialEmailIndicator` boolean  for each email entry that indicates whether it's official or not. This only matters as far as Harvard Key authentication is concerned, since the email will need to match and I think Harvard Key sends the official one.

Reply:
Ah, good to know about the `officialEmailIndicator`. Thanks!